src/components/LinkListItem.js

```diff
@@ -51,31 +50,59 @@ export default class LinkListItem extends React.Component {
             classes.push("has-right-icon");
         }
 
+        if (this.props.to.startsWith('mailto')) {
```

Comment:
Good idea. `<Link>` doesn't support setting `href` instead of `to` but I'll do it at the `LinkListItem` level instead.